cspell.yaml

```diff
@@ -48,6 +48,7 @@ words:
 - ratis
 - OM
 - SCM
+- DN
```

Comment:
Unlike OM and SCM we don't usually use `DN` as an abbreviation for datanode when talking about Ozone in an official context, so it was deliberately omitted here. It might be better to use an inline ignore to skip it in the mermaid code block.